docs/atlascli/command/atlas-clusters-create.txt

```diff
@@ -104,7 +104,7 @@ Options
    * - --tier
      - string
      - false
-     - Tier for each data-bearing server in the cluster. This value defaults to "M2".
+     - Tier for each data-bearing server in the cluster. To learn more about cluster tiers, see https://www.mongodb.com/docs/atlas/manage-clusters/#select-cluster-tier. This value defaults to "M2".
```

Comment:
I would add the learn more after the sentence about the default.

Reply:
Unfortunately I can't change the order because of the way this is generated (the default is added programmatically after the text from usage.go